src/api/user.js

```diff
@@ -40,6 +40,6 @@ export async function signup(url, body, callback) {
     const data = await response.json();
     callback(data);
   } catch (error) {
-    console.error('Error: ', error);
+    console.error('Error: ', error); // улучшить обработку ошибок
```

Comment:
Лучше оставлять 'TODO' - почти все IDE умеют с ними удобно работать)
А еще лучше не оставлять такое вообще)

```suggestion
    console.error('Error: ', error); // TODO: улучшить обработку ошибок
```

Reply:
хорошо) немного по другому теперь работаю с функцией и обрабатываю ошибку